client/core/wallet.go

```diff
@@ -55,6 +55,8 @@ type xcWallet struct {
 	hookedUp     bool
 	synced       bool
 	syncProgress float32
+
+	broadcasting *uint32
```

Comment:
Will you doc this field please?  I've been looking at core.go and I can't really tell the intend.  Let wallet state notes out?